src/context/mod.rs

```diff
@@ -652,8 +652,8 @@ impl<'a, T: This> CallContext<'a, T> {
     }
 
     /// Indicates the number of arguments that were passed to the function.
-    pub fn len(&self) -> i32 {
-        self.info.len(self)
+    pub fn len(&self) -> usize {
+        self.info.len(self) as usize
```

Comment:
Can you change this on the `CallbackInfo::len` and `sys::call::len` as well instead of casting?